### DISCUSSION TITLE
Not picking up on changes to omniDataTransforms, omniIntegrationProcedures and omniScripts

### DISCUSSION THREAD

**Discussion by pcjones10** [scolladon/sfdx-git-delta#826] [Q&A]
Hello @scolladon,
We added omniDataTransform, omniIntegrationProcedure and omniScript components to our manifest but not seeing changes to the components picked up.

**Comment by scolladon** [comment]
Hi @pcjones10 !

Please convert this discussion to an issue (follow the template and answer the questions as much as possible)
I'll look into it

**Reply by pcjones10** [reply]
@scolladon I have updated to Q&A.  Is that correct?  Is there any settings that I need to set in order to utilize newly introduced components?

**Reply by scolladon** [reply]
converting it into an [issue](https://github.com/scolladon/sfdx-git-delta/issues/new?assignees=scolladon&labels=bug&projects=&template=issue.md&title=%5BBUG+NAME%5D) would be best.
